Bottom: d71cb793d581ce43379c8128c346032eec774047
Top:    cbd69c3d7a7103a60a1d983b94facc35c2970e9f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-08 11:24:10 -0500

Refresh of all-docker_container

---
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index 73f762ae..10271473 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -140,7 +140,7 @@
     name: '{{ gitlab_db_container_name }}'
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
-    volumes-from: '{{ gitlab_db_dv_name }}'
+    volumes_from: '{{ gitlab_db_dv_name }}'
     image: '{{ gitlab_db_image_name }}:{{ docker_image_tag }}'
 
 - name: start container (gitlab)
@@ -155,7 +155,7 @@
       - '{{ gitlab_redis_container_name }}:gitlab-redis'
       - '{{ gitlab_db_container_name }}:gitlab-db'
       - '{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
-    volumes-from:
+    volumes_from:
       - '{{ openssl_dv_name }}'
       - '{{ gitlab_dv_name }}'
     env:
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 7df7d1af..45c1a895 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -152,7 +152,7 @@
     name: '{{ mantisbt_db_container_name }}'
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
-    volumes-from:
+    volumes_from:
       - '{{ mantisbt_db_dv_name }}'
     image: '{{ mantisbt_db_image_name }}:{{ docker_image_tag }}'
 
@@ -167,7 +167,7 @@
       -'{{ openldap_container_name }}:mantisbt_ldap'
       -'{{ mantisbt_db_container_name }}:mantisbt_db'
       -'{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
-    volumes-from: 
+    volumes_from: 
       - '{{ openssl_dv_name }}'
       - '{{ mantisbt_dv_name }}'
     env: 
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index c6e2e960..5128f7d2 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -123,7 +123,7 @@
     name: '{{ wiki_db_container_name }}'
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
-    volumes-from: '{{ wiki_db_dv_name }}'
+    volumes_from: '{{ wiki_db_dv_name }}'
     image: '{{ wiki_db_image_name }}:{{ docker_image_tag }}'
 
 - name: start container (mediwiki)
@@ -137,7 +137,7 @@
       - '{{ openldap_container_name }}:wiki_ldap'
       - '{{ wiki_db_container_name }}:wiki_db'
       - '{{ exim4_container_name }}:{{ container_addr_map.exim4.hostname }}'
-    volumes-from: 
+    volumes_from: 
       - '{{ openssl_dv_name }}'
       - '{{ wiki_dv_name }}'
       env:
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index e43e7de4..1ef879c3 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -110,7 +110,7 @@
     restart_policy: '{{ docker_restart_policy }}'
     hostname: '{{ container_addr_map.ldap.hostname }}'
     ports: '{{ container_port_map.ldap.port_args }}'
-    volumes-from:
+    volumes_from:
       - '{{ openssl_dv_name }}'
       - '{{ openldap_dv_name }}'
     env:
@@ -124,7 +124,7 @@
     restart_policy: '{{ docker_restart_policy }}'
     hostname: '{{ container_addr_map.ldap.hostname }}'
     ports: '{{ container_port_map.phpldapadmin.port_args }}'
-    volumes-from:
+    volumes_from:
       - '{{ openssl_dv_name }}'
     env:
       PHPLDAPADMIN_HTTPS: 'false'
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 8abf6024..5110de79 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -86,7 +86,7 @@
     ports: '{{ container_port_map.svn.port_args }}'
     link:
       - '{{ openldap_container_name }}:ldap'
-    volumes-from:
+    volumes_from:
       - '{{ openssl_dv_name }}'
       - '{{ svn_dv_name }}'
     env:
diff --git a/roles/docker-svn/tasks/restore.yml b/roles/docker-svn/tasks/restore.yml
index 5a8f10fd..f17b49bc 100644
--- a/roles/docker-svn/tasks/restore.yml
+++ b/roles/docker-svn/tasks/restore.yml
@@ -38,7 +38,7 @@
 - name: restore svn repositories
   docker_container:
     detach: False
-    volumes-from:
+    volumes_from:
       - '{{ svn_dv_name }}'
     volumes:
       - '{{ svn_docker_restore_dir }}:/tmp/import_export:z'
